src/main/java/de/tum/in/www1/artemis/domain/notification/SingleUserNotificationFactory.java

```diff
@@ -44,4 +48,40 @@ public static SingleUserNotification createNotification(Post post, NotificationT
         }
         return notification;
     }
+
+    /**
+     * Creates an instance of PlagiarismNotification
+     *
+     * @param plagiarismComparisonId ID of the plagiarism comparison associated with this case.
+     * @param recipient              affected student
+     * @param author                 instructor creating notification
+     * @param message                instructors message
+     * @return the PlagiarismNotification
+     */
+    private static SingleUserNotification createPlagiarismNotification(Long plagiarismComparisonId, Long courseId, User recipient, User author, String message, boolean isUpdate) {
+        String title = isUpdate ? "Update on your plagiarism case" : "Possible case of Plagiarism";
+        if (recipient.getLangKey() != null) {
+            Locale locale = Locale.forLanguageTag(recipient.getLangKey());
+            if (locale.equals(Locale.GERMAN) || locale.equals(Locale.GERMANY)) {
+                title = isUpdate ? "Update zu deinem Plagiatsfall" : "Möglicher plagiatsfall";
+            }
+        }
+        SingleUserNotification notification = new SingleUserNotification();
+        notification.setPriority(NotificationPriority.HIGH);
+        notification.setRecipient(recipient);
+        notification.setAuthor(author);
+        notification.setTarget(notification.targetForPlagiarismCase(plagiarismComparisonId, courseId));
+        notification.setNotificationDate(ZonedDateTime.now());
+        notification.setText(message);
+        notification.setTitle(title);
+        return notification;
+    }
+
+    public static SingleUserNotification createPlagiarismNotification(Long plagiarismComparisonId, Long courseId, User recipient, User author, String message) {
```

Comment:
Similar thing here. Please stick to the established style (even if it might not be the best but at least it is consistent).
If you compare this to the Single or GroupNotificationFactory every method there uses the NotificationType as a parameter and then sets the adequate properties based on this type.
My concrete recommendation: please stick to one method and e.g. switch-case set the different properties. (Just look at the existing implementation of the factories)